.ci.yaml

```
@@ -168,6 +188,56 @@ targets:
       target_file: ios_platform_tests.yaml
       package_sharding: "--shardIndex 3 --shardCount 4"
 
+  - name: Mac_arm64 ios_platform_tests_1_of_5 master
```

Now that you've felt the pain of `bringup`, you may be better off leaving off the number of shards so adding one doesn't require a rename of all of them, like:
```suggestion
  - name: Mac_arm64 ios_platform_tests_1 master
```

---

Good point. Keyong and I talked about the naming, but I don't think we talked about that specifically; I think including the number of shards was just a translation from the Cirrus behavior of the matrix variables being in the display name.

---

Renamed from `x_of_N` to `shard_x`. I also added `- plugins` while I'm renaming everything so the flutter/packages version can have no suffix to reduce work after the merge (currently they all have `- packages` to avoid name collisions in the central LUCI system).